release-notes.txt

```diff
@@ -1,3 +1,775 @@
+* gecko_sdk: update to v2.7.6
+* lwip:
+  * add IPv4 support for `netdev_tap`
+  * add support for nrf802154
+  * enable IPv4/IPv6 dual stack mode
+  + lwip_sock: provide implementation for `sock_*_recv_buf()`
+- lua: remove unnecessary patch
++ paho-mqtt: add package and example
+* nimble:
+  + add static connection manager `statconn`
+  * netif:
+    * set host thread prio to 1
+    * simplify mbuf pool initialization
+  * update NimBLE ver (cd7d7aa286) + timing param cleanup
++ openwsn: re-integrate the network stack as a package
+* tinydtls:
+  + add `sock_async` support for `sock_dtls`
+  + use latest development version (eda63f000c)
+* ucglib: refactoring and fixes
+
+Boards
+------
++ cc1352p-launchpad: add support
+* common/blxxxpill:
+  + provide periph_rtt
+  * improve adc
++ dwm1001: add support to Decawave DWM1001 dev board
++ mcb2388: misc enhancements
++ nrf52840-dongle: add support
++ nrf52840dk: Add MTD configuration
++ nucleo-g474re: add support
+* pic32-clicker: rename to 6lowpan-clicker
++ samr30-xpro: add SPI dev for ext pin header
++ samr3x-xpro: add riotboot feature
++ serpente: add support
++ sltb001a: enable CCS811 sensor
++ sodaq-sff: add support for SODAQ SFF
++ stm32f429i-disco: add support
++ weact-f411ce: add support
++ wemos-zero: add support
+
+CPU
+---
+* arm7_common: Update to inlineable IRQ API
+* atmega_common: Update to inlineable IRQ API
+* cortexm:
+  + Allow to define reserved fields in CortexM vector table
+  * "free" SVC
+* cc26x2_cc13x2:
+  * add oscillator switching functions
+  * add periph_i2c implementation
+  * trim device registers on `cpu_init`
+* esp32:
+  * support of multiple heaps for newlib malloc
+  * replace RTC implementation by RTT implementation
+* fe310: migrate to inlined IRQ API
++ kinetis: implement power modes for pm_layered
+* lpc2387:
+  * PM: enable SLEEP & POWERDOWN mode
+  * timer: use lpc2387_pclk_scale()
++ mips_pic32_common: Implement GPIO IRQ
++ native:
+  + add periph_eeprom driver implementation
+  * rename trace -> backtrace
++ nrf52:
+  + add black magic probe config
+  + add missing vendor files
+  + implement EasyDMA-based SPI peripheral implemenation
++ nrf52811: initial support
+* nrf5x_common:
+  * add configurable RTT_FREQUENCY
+  * enable bias correction in hwrng
+* sam0_common:
+  + add DMA peripheral driver
+  + add support for saml21, samd5x
+  + implement periph/dac
+  * Make SPI peripheral DMA compatible
+  * move PWM to common code
+  * spi: move clk pin muxing into spi_acquire / spi_release
+  + UART: implement inverted RX & TX
+* samd21: PWM don't hard-code number of channels to 3
+* samd5x: allow to use XOSC as clock source
+* saml1x: pm: set deep flag
++ stm32:
+  + add QDEC support for CPU_FAM_STM32F1
+  + add non-blocking UART
+  + implement reset to bootloader
+  * filter availability of RNG by CPU not by board
+  * stm32_common: make RTT_FREQUENCY configurable
+  + stm32f4: Add DMA config for SPI
+  + stm32g4: add support
```

Comment:
I'm missing something along the lines of
```
  + Decrease latency in SPI DMA path
```

Something that describes the effect of both https://github.com/RIOT-OS/RIOT/pull/14096 and https://github.com/RIOT-OS/RIOT/pull/14097 in a single note. Feel free to reword my suggestion to something that makes sense for the reader

Reply:
See my latest change.